src/js/lib/DOMTranslation.ts

```diff
@@ -974,7 +976,7 @@ class DOMTranslation {
       }
     }
     for (const range of translationRanges) {
-      range.visibleInCurrentView = DOMExtensions.elementIsVisible(range.startMarker, this.registredIframes)
+      range.visibleInCurrentView = this.pluginOptions.translation.translateWholePage ? true : DOMExtensions.elementIsVisible(range.startMarker, this.registredIframes)
```

Comment:
Šis ir vajadzīgs arī pie pēdējām izmaiņām?

Reply:
nope, nav vajadzības, izņēmu